contracts/utils/Base64.sol

```diff
@@ -22,7 +22,7 @@ library Base64 {
     }
 
     /**
-     * @dev Converts a `bytes` to its Bytes64Url `string` representation.
+     * @dev Converts a `bytes` to its Bytes64Url `string` representation. Skips `=` padding.
```

Comment:
```suggestion
     * @dev Converts a `bytes` to its Bytes64Url `string` representation. Output is not padded with `=` as authorized 
     * by https://www.rfc-editor.org/rfc/rfc4648[rfc4648].
```

Reply:
changed `authorized` to `specified` since I think its meaning is more accurate in this context